shared/js/otSdkDriver.js

```diff
@@ -776,6 +789,21 @@ loop.OTSdkDriver = (function() {
     },
 
     /**
+     * Sends the cursor position on the data channel.
+     *
+     * @param {String} message The message to send.
+     */
+    sendCursorMessage: function(message) {
+      if (!this._publisherChannel || !this._subscriberChannel) {
+        return;
```

Comment:
also, why checking for `_subscriberChannel` if we don't use it later? is it the default if primary fails?